src/main/kotlin/net/maizegenetics/phgv2/pathing/FindPaths.kt

```diff
@@ -220,11 +218,13 @@ class DiploidPathFinding: CliktCommand(help = "Impute best diploid path using re
         for (imputeJob in jobList) imputeJob.join()
         pathChannel.close()
 
+        saveJob.join()
```

Comment:
Was this what was causing the issues with the failing unit tests?  It didn't properly block when it was saving?

Reply:
I finally figured it out. The read mappings is generated each time and has a random element. Every so often, it generates a dataset that imputes an incorrect haplotype. I increased the number of reads per haplotype, which seems to take care of that.